e2e/support/helpers/e2e-viz-settings-helpers.js

```diff
@@ -13,3 +13,10 @@ export function openSeriesSettings(field, isBreakout = false) {
       .click();
   }
 }
+
+export function openVizType(tabName = "Chart") {
+  cy.findByTestId("viz-settings-button").click();
```

Comment:
Design feedback: my OCD kicks in hard when I see:
- that there's one button with a different `border-radius` on this page
- that the most important button (i.e. "Visualize", since it kind of submits the form) has the lowest height of all buttons on this page

![image](https://github.com/user-attachments/assets/32f8b57f-17eb-4f60-9145-2c13f6146de3)